sepolicy/file_contexts

```diff
@@ -0,0 +1,2 @@
+/vendor/bin/hw/android.hardware.gnss@1.1-service.xenvm                          u:object_r:hal_gnss_default_exec:s0
```

Comment:
You've assigned 'android.hardware.gnss@1.1-service.xenvm' to the 'hal_gnss_default_exec'. 

But I do not see additional rights, which are required for our implementation of GNSS HAL:

```
typeattribute hal_gnss_default hal_automotive_socket_exemption;
allow hal_gnss_default vndbinder_device:chr_file { ioctl open read write };
allow hal_gnss_default hidl_base_hwservice:hwservice_manager add;
allow hal_gnss_default hwservicemanager_prop:file { getattr map open read };
allow hal_gnss_default fwmarkd_socket:sock_file write;
allow hal_gnss_default self:tcp_socket { create getopt read setopt write };

userdebug_or_eng(`
    dontaudit hal_gnss_default netd:unix_stream_socket connectto;
')
```
----

Also, maybe we should allow GNSS to read the 'vendor_vis_prop'?

```
get_prop(hal_gnss_default, vendor_vis_prop)
```

Reply:
Done.